package com.stefanenko.gitphone.util.exception

object DataBaseExceptionConstantStorage {
    const val INSERT_ERROR = "INSERT_ERROR"
    const val DELETE_ERROR = "DELETE_ERROR"
    const val DELETE_ERROR_UNMATCHED_ID = "DELETE_ERROR_UNMATCHED_ID"
    const val NO_SUCH_USER_IN_DATABASE = "NO_SUCH_USER_IN_DATABASE"
}